desktop/libs/notebook/src/notebook/conf.py

```diff
@@ -152,6 +157,16 @@ def get_ordered_interpreters(user=None):
     for i in interpreters
   ]
 
+def computes_for_dialect(dialect, user):
+  from beeswax.models import Namespace
```

Comment:
why here? is it due to Cyclic dependency? if Yes can you please add a comment # Cyclic dependency after import.

Reply:
yes, this is due to cyclic deps. Will add a comment.